doozer/doozerlib/comment_on_pr.py

```diff
@@ -1,11 +1,15 @@
 from ghapi.all import GhApi
 from doozerlib.pushd import Dir
 from dockerfile_parse import DockerfileParser
+from doozerlib.constants import BREWWEB_URL
 
 
 class CommentOnPr:
-    def __init__(self, distgit_dir: str, token: str):
+    def __init__(self, distgit_dir: str, token: str, nvr: str, build_id: str, distgit_name: str):
```

Comment:
`token` is just `os.getenv(constants.GITHUB_TOKEN)`, so it can be inferred locally and not passed in as a constructor parameter